src/syntax-highlight/language-ext.js

```diff
@@ -52,6 +52,8 @@ module.exports = {
     '.styl': 'language-stylus',
     '.svcmap': 'language-markup',
     '.tpl': 'language-smarty',
+    '.t': 'language-perl',
+    '.tt': 'language-markup'
```

Comment:
Seems like you missed a comma here! 😅  Also, can you order them alphabetically? It would be .t, .tpl then .tt I believe